base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -20,8 +20,6 @@
 namespace base_local_planner {
 
 LatchedStopRotateController::LatchedStopRotateController(const std::string& name) {
-  ros::NodeHandle private_nh("~/" + name);
-  //private_nh.param("latch_xy_goal_tolerance", latch_xy_goal_tolerance_, false);
   latch_xy_goal_tolerance_ = false;
```

Comment:
should this be true so that it defaults to latching behavior?

Reply:
Yeah, that's fair.